src/serverdlg.cpp

```diff
@@ -106,16 +106,16 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
     edtLocationCity->setAccessibleName ( tr ( "City where the server is located line edit" ) );
 
     // location country
-    QString strLocCountry = "<b>" + tr ( "Location country" ) + ":</b> " +
-                            tr ( "The country in "
-                                 "which this server is located can be set here. If a country is "
+    QString strLocCountry = "<b>" + tr ( "Country/Region" ) + ":</b> " +
+                            tr ( "The country or region in "
+                                 "which this server is located can be set here. If a location is "
                                  "entered, it will be shown in the connect dialog server list at the "
                                  "clients." );
 
     lblLocationCountry->setWhatsThis ( strLocCountry );
     cbxLocationCountry->setWhatsThis ( strLocCountry );
 
-    cbxLocationCountry->setAccessibleName ( tr ( "Country where the server is located combo box" ) );
+    cbxLocationCountry->setAccessibleName ( tr ( "Combo box for location of this server" ) );
```

Comment:
Best practice for copy is not to refer to UI or positional elements as they might change (and in this case people might not know what a "combo box" is). Can we just say this instead? I don't know where this text appears exactly though.

```suggestion
    cbxLocationCountry->setAccessibleName ( tr ( "The location of this server" ) );
```

Reply:
These strings set with `setAccessibleName()` are for screen readers, they are actually not displayed on the UI.

Reply:
Most of the accessibility string state the nature of the UI component that they are describing - it helps the visually impared handle the screen navigation if they're prompted in this way.